cmd/cli/main.go

```diff
@@ -48,6 +50,17 @@ var (
 	toUser   = flag.String("to-user", "", "Transfer subscriptions and contacts to user.")
 )
 
+var (
+	pullTrigger            = flag.String("pull-trigger", "", "Get trigger from redis and save it to file")
+	pullTriggerMetrics     = flag.String("pull-trigger-metrics", "", "Get trigger patterns and metrics from redis and save it to file")
+	pushTrigger            = flag.Bool("push-trigger", false, "Get trigger in JSON from file and save it to redis")
+	pushTriggerMetricsByID = flag.String("push-trigger-metrics", "", "Get trigger patterns and metrics in JSON from stdin and save it to redis")
+	pushTriggerDump        = flag.Bool("push-trigger-dump", false, "Get trigger dump in JSON from stdin and save it to redis")
+	triggerFile            = flag.String("trigger-file", "", "File that holds trigger JSON")
+	triggerMetricsFile     = flag.String("trigger-metrics-file", "", "File that holds trigger metrics JSON")
+	triggerDumpFile        = flag.String("trigger-dump-file", "", "File that holds trigger dump JSON from api method response")
+)
+
```

Comment:
Why do we need to have all this options? As I mentioned: the use case is a simple: if something went wrong with trigger check. We just go to API and dump all data about trigger, metrics, and retentions in one JSON. Then we need only one method in CLI that will place all this data into Redis.

Reply:
fixed